pennylane/tensorbox/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This package contains the ``TensorBox`` class, a container for array-like objects
+that allows array manipulation to be performed in a unified manner for supported
+linear algebra packages.
```

Comment:
Shall we mention that it is loosely inspired by eagerpy?

Reply:
I found that this docstring actually doesn't make it clear enough what it is doing. Could we mention that this class handles the dispatching of function calls to the relevant libraries?

Reply:
I've re-written this docstring, and replaced all 'linear algebra packages' with 'tensor/array manipulation frameworks' :)